blocks/hooks/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * WordPress dependencies
  */
-import { createHooks } from '@wordpress/hooks';
+import * as hooks from '@wordpress/hooks';
```

Comment:
Should we drop this and just import it when it's used (on each of the hooks files)?

Reply:
We could. I need to play with unit tests, but given that they are isolated, we should be good.